### ISSUE TITLE
PR into teammate's branch aren't factored into master reposense and individual tp Dashboard progress, will this affect grading in anyway?

### ISSUE THREAD

**Issue by SeeYangZhi** [nus-cs2103-AY2425S1/forum#507]
PR into teammate's branch isn't factored into master reposense (but that PR is subsequently merged into master) in the code dashboard and individual tp progress Dashboard, will this affect grading in any way?

E.g. https://github.com/AY2425S1-CS2103-F13-1/tp/pull/122

**Comment by damithc** [comment]
@SeeYangZhi The code will be recognized once it is merged to the master branch. IIRC, PRs will be recognized too (you can cross-check against the PR count shown in the tP dashboard).
That said, note the [following FAQ in Week 8](https://nus-cs2103-ay2425s1.github.io/website/schedule/week8/project.html):

![image](https://github.com/user-attachments/assets/ce32a755-cac6-4fdb-837c-1d41e66ab9a4)
